br/tests/br_full/run.sh

```diff
@@ -104,6 +104,37 @@ for ct in limit lz4 zstd; do
   fi
 done
 
+for granularity in "fine-grained" "coarse-grained"; do
+  for i in $(seq $DB_COUNT); do
+      run_sql "DROP DATABASE $DB${i};"
+  done
+
+  # restore full
+  echo "restore with $ct backup start..."
+  export GO_FAILPOINTS="github.com/pingcap/tidb/br/pkg/restore/restore-storage-error=1*return(\"connection refused\");github.com/pingcap/tidb/br/pkg/restore/restore-gRPC-error=1*return(true)"
+  export GO_FAILPOINTS=$GO_FAILPOINTS";github.com/pingcap/tidb/br/pkg/restore/no-leader-error=3*return(true)"
```

Comment:
This will prevent `GO_FAILPOINTS` get split once there are some spaces in it.

```suggestion
  export GO_FAILPOINTS="${GO_FAILPOINTS};github.com/pingcap/tidb/br/pkg/restore/no-leader-error=3*return(true)"
```